bika/lims/content/labcontact.py

```diff
@@ -63,11 +63,6 @@
             description=_("The laboratory department"),
         ),
     ),
-    atapi.ComputedField('DepartmentTitle',
-                        expression="context.getDepartment() and context.getDepartment().Title() or ''",
-                        widget=atapi.ComputedWidget(
-                            visible=False,
-                        )),
     atapi.ReferenceField('Departments',
```

Comment:
This is wrong. Although I know Department from LabContact will be deleted soon (there is already a many-to-many relation in labcontact), this removal shouldn't be in this Pull Request. The scope of this PR is Analysis Category.